.github/workflows/test.yml

```diff
@@ -239,9 +239,60 @@ jobs:
 
       - name: Run tests
         run: |
-          nox -s setup test report --no-error-on-missing-interpreters -- --confcutdir . tests/integrations/postgresql
+          coverage run -m nox -s databases -- --coverage --databases=postgresql
         env:
-          PRISMA_PY_POSTGRES_URL: postgresql://postgres:postgres@localhost:5432/postgres
+          COVERAGE_FILE: .cache/.coverage
+          POSTGRESQL_URL: postgresql://postgres:postgres@localhost:5432/postgres
+
+      - name: Combine coverage
+        run: |
+          nox -s report
+
+      - name: Upload coverage
+        run: |
+          codecov --file coverage.xml
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+  sqlite:
+    name: SQLite
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -U -r pipelines/requirements/ci.txt .[dev]
+
+      - uses: actions/cache@v3
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+
+      # TODO: cache prisma engines
```

Comment:
That would really help with the CI time, but feel free to spin this out into another ticket and I would be happy to pick it up. I think you had a running TODO list ticket to chuck it into

Reply:
Good point - #550.